libs/output-mapping/tests/Storage/BucketCreatorTest.php

```diff
@@ -8,42 +8,42 @@
 use Keboola\OutputMapping\Storage\BucketCreator;
 use Keboola\OutputMapping\SystemMetadata;
 use Keboola\OutputMapping\Tests\AbstractTestCase;
-use Keboola\OutputMapping\Tests\Needs\NeedsRemoveBucket;
-use Keboola\OutputMapping\Tests\Writer\CreateBranchTrait;
+use Keboola\OutputMapping\Tests\Needs\NeedsDevBranch;
+use Keboola\OutputMapping\Tests\Needs\NeedsEmptyInputBucket;
 use Keboola\OutputMapping\Writer\Table\MappingDestination;
-use Keboola\StorageApiBranch\ClientWrapper;
-use Keboola\StorageApiBranch\Factory\ClientOptions;
 
 class BucketCreatorTest extends AbstractTestCase
 {
-    use CreateBranchTrait;
-
-    #[NeedsRemoveBucket('in.c-unexistsBucket')]
+    #[NeedsEmptyInputBucket]
     public function testEnsureDestinationBucket(): void
     {
+        $this->clientWrapper->getTableAndFileStorageClient()->dropBucket($this->emptyInputBucketId);
```

Comment:
Bucket je unikatni pro test. Predtim mel fixni nazev.

Resim to zatim takhle, ze ho hned na zacatku testu dropnu. Abych mel naplneny ty promenne s jeho ID.
Mozna bych upravil `NeedsEmptyInputBucket` aby mel property na dry run, ze by to naplnilo jen ty promenne ale bucket by to nezakladalo a pokud existuje, tak ho smazalo 🤔

Reply:
Ale když nebude existovat, tak to failne, ne?

Reply:
Existovat bude, protoze tu je `NeedsEmptyInputBucket`